Tests/SonarScanner.MSBuild.Shim.Test/AdditionalFilesServiceTest.cs

```diff
@@ -41,15 +42,44 @@ public AdditionalFilesServiceTest()
     }
 
     [TestMethod]
-    public void AdditionalFiles_NoExtensionsFound()
+    public void AdditionalFiles_EmptyServerSettings_NoExtensionsFound()
     {
-        var files = sut.AdditionalFiles(new() { LocalSettings = [], ServerSettings = [] }, directoryInfo);
+        var files = sut.AdditionalFiles(new() { ServerSettings = [] }, directoryInfo);
+
+        files.Sources.Should().BeEmpty();
+        files.Tests.Should().BeEmpty();
+        wrapper.DidNotReceive().EnumerateFiles(Arg.Any<string>(), Arg.Any<string>(), Arg.Any<SearchOption>());
```

Comment:
NIT: I sligthly easier version of this is:
```suggestion
        wrapper.DidNotReceiveWithAnyArgs().EnumerateFiles(default, default, default);
```